generators/app/templates/app/controllers/admin/Profile.ts

```diff
@@ -3,7 +3,11 @@ import { ModelAdminController } from "@/libraries/ModelAdminController";
 import { Profile } from "@/models/Profile";
 
 @Controller("profile", Profile)
-export class ProfileAdminController extends ModelAdminController<Profile> {}
+export class ProfileAdminController extends ModelAdminController<Profile> {
+  constructor() {
```

Comment:
If we are not doing anything special in the constructor, we can remove it, right?

Reply:
@rodksquare  Yes, you are right for some reason my IDE was throwing error if it did not put the constructor to him. I´lss change in all the classes.